great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -266,8 +257,14 @@ def test_connection(self) -> None:
         Raises:
             TestConnectionError: If the connection test fails.
         """
-        if not self._data_connector.test_connection():
-            raise TestConnectionError(self._test_connection_error_message)
+        try:
+            if self._data_connector.test_connection():
+                return None
```

Comment:
@Kilo59 There seems to be an. opportunity to make this more elegant: if fail `test_connection()`, then raise; otherwise, we are at the method's end.  Thanks.